sources/ReplicantWindow.cpp

```diff
@@ -7,16 +7,20 @@
  */
 
 
+#include <Catalog.h>
 #include <ControlLook.h>
 #include <LayoutBuilder.h>
 
 #include "DropZoneTab.h"
 #include "ReplicantWindow.h"
 
+#undef B_TRANSLATION_CONTEXT
+#define B_TRANSLATION_CONTEXT "ReplicantWindow"
+
 
 ReplicantWindow::ReplicantWindow(BRect frame)
 	:
-	BWindow(BRect(0, 0, 120, 100), ("Resize"),
+	BWindow(BRect(0, 0, 120, 100), (B_TRANSLATE("Resize")),
```

Comment:
The length of this window title determines the minimal width of the Replicant.
I suggest to simply use "Size" which might be easier translated than a verb. In German, for exampl, there's not really one word for it... "Größe ändern". So maybe:
> BWindow(BRect(0, 0, 120, 100), (B_TRANSLATE_COMMENT("Size", "As label of the Dropzone Replicant window, the translation should be as short as possible.")),